components/raftstore/src/store/util.rs

```diff
@@ -745,6 +751,139 @@ pub(crate) fn u64_to_timespec(u: u64) -> Timespec {
     Timespec::new(sec as i64, nsec as i32)
 }
 
+// ParsedEntry wraps raft-proto `Entry` and used to avoid parsing raft command
+// from entry's data repeatedly. The parsed command may be used in multiple
+// places, so cache it at the first place.
+pub struct ParsedEntry {
```

Comment:
consider a better name? maybe as use `Entry` , `RaftEntry` or `LogEntry` directly?

Reply:
These names can't be distinguished with the current `raftpb::Entry`

Reply:
It's a little strange with parsed.

Reply:
maybe call it `LazyEntry`?

Reply:
`Parsed` describes the behavior, that it provides parsed items (cmd and conf_change) to outers. `parse` is just an inner function.

Reply:
What about `PlainEntry` or `DecodedEntry`?

Reply:
`Entry` is defined in proto, I can't change the name.